src/style.css

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: Avenir;
+  src: local("Avenir"), url("./fonts/Avenir-Heavy.ttf");
+  src: local("Avenir"), url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  color: #3d4e61;
+  padding: 0;
+  margin: 0;
+}
+
+.forms-page__container {
+  max-width: 1030px;
+  margin: 0 auto;
+}
+
+.forms-page__form-big {
+  display: flex;
+  margin: 20px 8px;
+  height: 70px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  background: url("./images/Search.svg") 26px 49% no-repeat;
+  transition: box-shadow 0.3s ease 0s;
+}
+
+.forms-page__form-big:hover {
+  box-shadow: 0  3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.forms-page__input-big {
+  width: 100%;
+  background-color: transparent;
+  padding-left: 62px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  outline: 0;
+
+  font-family: Avenir, sans-serif;
```

Comment:
you may set it for the entire page and inherit
net critical here since on the page only these 2 inputs exist, anyway - FYI